mwdb/web/src/components/Comments.js

```diff
@@ -10,215 +10,170 @@ import { ObjectContext } from "@mwdb-web/commons/context";
-                                    ? [value, <br/>]
-                                    : value)
+                        props.removeComment &&
+                        <button className="btn btn-link p-0 remove-comment-link"
+                                onClick={() => props.removeComment(props.id)}>Remove</button>
                     }
-                    </span>
-                    <p>
-                        {
-                            this.props.removable &&
-                            <button className="btn btn-link p-0 remove-comment-link"
-                                    onClick={() => this.props.removeComment(this.props.id)}>Remove</button>
-                        }
-                    </p>
-                </div>
-                <p className="float-right">
-                    <small>{readableTime(new Date(this.props.timestamp))}</small>
                 </p>
-            </li>
-
-        );
-    }
+            </div>
+            <p className="float-right">
+                <small>{readableTime(new Date(props.timestamp))}</small>
+            </p>
+        </li>
+    );
 }
 
+function CommentList(props) {
+    const commentNodes = props.data
+        .map((comment, index) => (
+            <Comment id={comment.id}
+                     comment={comment.comment}
+                     author={comment.author}
+                     timestamp={comment.timestamp}
+                     removeComment={props.removeComment}
+                     key={index}
+            />
+        ));
+    return (
+        <div className="commentList">
+            {
+                commentNodes.length > 0 ? (
+                    <ul className="list-group list-group-flush">
+                        {commentNodes}
+                    </ul>
+                    ) : (
+                        <div className="text-muted">No comments to display</div>
+                )
+            }
+        </div>
+    );
+}
 
-class CommentBox extends Component {
-    itemsCountPerPage = 15;
-
-    state = {
-        comments: [],
-        activePage: 1,
-        modalIsOpen: false
-    }
-
-    static contextType = ObjectContext;
-
-    openModal(comment_id) {
-        this.setState({modalIsOpen: true, commentToRemove: comment_id});
-    };
+function CommentForm(props) {
+    const [text, setText] = useState("")
+    return (
+        <form className="commentForm"
+              onSubmit={
+                  (ev) => {
+                      ev.preventDefault();
+                      props.submitComment(text);
+                      setText("");
+                  }
+              }>
+            <div className="input-group">
+                <textarea className="form-control" placeholder="Say something..."
+                          value={text}
+                          onChange={ev => setText(ev.target.value)}
+                          onKeyDown={evt => evt.ctrlKey && evt.keyCode === 13 && props.submitComment(text)}/>
```

Comment:
It would be nice to have helper function that calls submitComment and setText like `onSubmit` does. The behavior of both submission methods should be consistent.